hw2_2/hw2-2/Program.cs

```diff
@@ -0,0 +1,298 @@
+using System;
+using System.Collections.Generic;
+
+namespace hw2_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            StackCalculatorOfReversePolishNotation calculator = new();
+            bool correctNotation;
+            Console.WriteLine(
+                "This program calculates expression in reverse polish notation by stack, you can choose one of the options:\n" +
+                "1. Stack on array base\n" +
+                "2. Stack on list base\n");
+            Console.Write("write number 1 or 2: ");
+            string choice = Console.ReadLine();

```

Comment:
```suggestion
            string choice = Console.ReadLine().Trim();
```
Что пробелы игнорировать.
Только вызывать надо чуть иначе, учитывая, что применяем не к строке, а к `string?`.

Reply:
Тип результата `ReadLine()` - это `string?` и работать с ним надо соответственно.